.github/workflows/release.yaml

```diff
@@ -0,0 +1,43 @@
+on:
+  push:
+    tags:
+      - 'release/*'
+
+name: Release
+defaults:
+  run:
+    shell: pwsh
```

Comment:
why do you need that?

Reply:
don't know, this was copied from jetfire-cli repo. Removed it